keycloak.yaml

```diff
@@ -62,6 +58,13 @@ subpackages:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"/opt
           ln -s /usr/share/java/keycloak "${{targets.subpkgdir}}"/opt/
+  - name: keycloak-scripts
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/bin
+          for i in kc.sh kcadm.sh kcreg.sh; do
+            ln -sf /usr/share/java/keycloak/bin/$i ${{targets.subpkgdir}}/usr/bin/$i
```

Comment:
I think we want `keycloak-scripts` to contain the actual files, not just symlinks, right?

Reply:
yes, but I think since in operator manifest we are going to use our keycloak image as well so probably that can handle that?:

```yaml
    spec:
      containers:
        - env:
            - name: RELATED_IMAGE_KEYCLOAK
              value: quay.io/keycloak/keycloak:24.0.3
          image: quay.io/keycloak/keycloak-operator:24.0.3
  ```

Reply:
the other option is to add scripts it in `keycloak-operator` itself probably and symlink or cp it there itself without calling `keycloak-scripts`?

Reply:
> yes, but I think since in operator manifest we are going to use our keycloak image as well so probably that can handle that?

I would avoid touching `keycloak` at all and just copy these scripts into `keycloak-scripts` so that `keycloak-operator` can depend on them.